src/backy/revision.py

```diff
@@ -12,17 +12,23 @@
 TRUST_DISTRUSTED = 'distrusted'
 TRUST_VERIFIED = 'verified'
 
+TAG_MANUAL_PREFIX = "manual:"
+
 
 logger = logging.getLogger(__name__)
 
 
+def filter_schedule_tags(tags):
+    return {t for t in tags if not t.startswith(TAG_MANUAL_PREFIX)}
+
+
 class Revision(object):
 
     uuid = None
     timestamp = None
     parent = None
     stats = None
-    tags = ()
+    tags = set()
```

Comment:
Danger, Will Robinson!

Adding mutable objects on class attributes, will result in inplace-modifications (instead of reassignments) being shared between instances!

Reply:
So, this sounds like a perfect opportunity to start using type annotations and add mypy checking to the pre-commit hook.